fastdds_python/test/CMakeLists.txt

```diff
@@ -15,18 +15,12 @@
 # Compile types
 add_subdirectory(types)
 
-set(fastcdr_version_argument "v2")
-if (${fastcdr_VERSION_MAJOR} EQUAL 1)
-    set(fastcdr_version_argument "v1")
-endif()
-
-
 add_test(NAME api_tests
     COMMAND
         ${Python3_EXECUTABLE}
         -m pytest
         -vrP
-        --cdr ${fastcdr_version_argument}
+        --cdr v2
```

Comment:
`--cdr v2` is not necesary. We should remove it. Also all code depending in this argument. For example:
- This file https://github.com/eProsima/Fast-DDS-python/blob/feature/remove_fastcdrv1_code/fastdds_python/test/api/conftest.py
- This kind of functions https://github.com/eProsima/Fast-DDS-python/blob/feature/remove_fastcdrv1_code/fastdds_python/test/api/test_datareader.py#L10
- and it usage https://github.com/eProsima/Fast-DDS-python/blob/feature/remove_fastcdrv1_code/fastdds_python/test/api/test_datareader.py#L116